src/main/scala/tile/LazyRoCC.scala

```diff
@@ -182,26 +183,27 @@ class AccumulatorExampleModuleImp(outer: AccumulatorExample)(implicit p: Paramet
   io.mem.req.bits.tag := addr
   io.mem.req.bits.cmd := M_XRD // perform a load (M_XWR for stores)
   io.mem.req.bits.size := log2Ceil(8).U
-  io.mem.req.bits.signed := Bool(false)
-  io.mem.req.bits.data := Bits(0) // we're not performing any stores...
-  io.mem.req.bits.phys := Bool(false)
+  io.mem.req.bits.signed := false.B
+  io.mem.req.bits.data := 0.U // we're not performing any stores...
+  io.mem.req.bits.phys := false.B
   io.mem.req.bits.dprv := cmd.bits.status.dprv
 }
 
 class  TranslatorExample(opcodes: OpcodeSet)(implicit p: Parameters) extends LazyRoCC(opcodes, nPTWPorts = 1) {
   override lazy val module = new TranslatorExampleModuleImp(this)
 }
 
+@chiselName
 class TranslatorExampleModuleImp(outer: TranslatorExample)(implicit p: Parameters) extends LazyRoCCModuleImp(outer)
     with HasCoreParameters {
-  val req_addr = Reg(UInt(width = coreMaxAddrBits))
-  val req_rd = Reg(io.resp.bits.rd)
+  val req_addr = Reg(UInt(coreMaxAddrBits.W))
+  val req_rd = Reg(io.resp.bits.rd.cloneType)
```

Comment:
I think use of `cloneType` is discouraged in favor of `chiselTypeOf`.

```suggestion
  val req_rd = Reg(chiselTypeOf(io.resp.bits.rd))
```